### PR TITLE
Lost Kingdoms

### PR DESCRIPTION
**Pull request by NineKain**
I made one profile for Lost Kingdoms

### CODE REVIEW

File: Lost Kingdoms/UniversalDIT_Flat_Pressed.json [hunk 1]
```diff
@@ -0,0 +1,2136 @@
...
+      "`Axis 0-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Left Stick.png",
+      "THUMBL": "../#DefaultDevices/DualShock4/Flat_Pressed/L3.png",
+      "`Axis 4+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 4-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3+`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "`Axis 3-`": "../#DefaultDevices/DualShock4/Flat_Pressed/Right Stick.png",
+      "THUMBR": "../#DefaultDevices/DualShock4/Flat_Pressed/R3.png"
+    }
+  },
+  "generated_folder_name": "#Generated_",
```
Comment (Venomalia) [inline Lost Kingdoms/UniversalDIT_Flat_Pressed.json:2117]: You must also set `generated_folder_name` for **UniversalDIT_Flat_Pressed** to `Generated_LostKingdoms`.

**Comment by Venomalia** [comment]
Good work, have you tested the pack?
It looks good so far, you just have to set the `generated_folder_name` in the **UniversalDIT_Flat_Pressed.json**.

**Comment by NineKain** [comment]
Just did the change you suggested and yeah I have tested the pack, works fine on my end

File: Lost Kingdoms/UniversalDIT_Flat_Pressed.json [hunk 2]
```diff
@@ -2114,7 +2114,7 @@
       "THUMBR": "../#DefaultDevices/DualShock4/Flat_Pressed/R3.png"
     }
   },
-  "generated_folder_name": "#Generated_",
+  "Generated_LostKingdoms": "#Generated_",
```
Comment (Venomalia) [inline Lost Kingdoms/UniversalDIT_Flat_Pressed.json:2117]: The line must be like this.
`  "generated_folder_name": "#Generated_LostKingdoms",`

**Pull request merged**